.github/workflows/ios.yaml

```diff
@@ -8,11 +8,11 @@ jobs:
   build:
     uses: ./.github/workflows/setup_macos.yaml
     with:
-      gradleTask: compileKotlinIosX64
+      gradleTask: compileKotlinIos${{ runner.arch == 'X86' && 'X86' || runner.arch == 'X64' && 'x86_64' || runner.arch == 'ARM' && 'SimulatorArm32' || 'SimulatorArm64' }}
```

Comment:
There is no `SimulatorArm32`

But honestly, don't even bother detecting 32 bit.